pageserver/src/billing_metrics.rs

```diff
@@ -91,6 +94,7 @@ impl FromStr for BillingMetricKind {
             "synthetic_storage_size" => Ok(Self::SyntheticStorageSize),
             "resident_size" => Ok(Self::ResidentSize),
```

Comment:
Sure. Thank you for pointing out.

It was changed when on-demand download PR landed, because physical size has different meaning there. I was pretty sure that we've synced with billing team on that, but probably missed the spec part